sdks/python/setup.py

```diff
@@ -142,11 +142,11 @@ def get_version():
     'proto-google-cloud-datastore-v1>=0.90.0,<=0.90.4',
     # [BEAM-4543] Datastore IO is not supported in Python 3.
     'googledatastore>=7.0.1,<7.1; python_version < "3.0"',
-    'google-cloud-pubsub==0.39.0',
+    'google-cloud-pubsub>=0.39.0',
     # GCP packages required by tests
     'google-cloud-bigquery>=1.6.0,<1.7.0',
-    'google-cloud-core==0.28.1',
-    'google-cloud-bigtable==0.31.1',
+    'google-cloud-core>=0.28.1',
```

Comment:
Same as above, and can we move the test dependencies to `REQUIRED_TEST_PACKAGES`? So that users won't need to install the test deps, and we have less chance to introduce new conflicts with the cloud client libraries.

Reply:
google-cloud-bigquery might be the only test dependency. (cc: @markflyhigh)